contracts/chain-adapters/l2/WithdrawalHelperBase.sol

```diff
@@ -86,8 +99,19 @@ abstract contract WithdrawalHelperBase is CircleCCTPAdapter, MultiCaller {
     ) public virtual;
 
     /*
-     * @notice Checks that the L1 msg.sender is the `HUB_POOL` address.
+     * @notice Checks that the L1 msg.sender is the `CROSS_DOMAIN_ADMIN` address.
```

Comment:
ooc, why is `CROSS_DOMAIN_ADMIN` all caps?

Reply:
It shouldn't be... This was an artifact of before it was a proxy. Fixed here https://github.com/across-protocol/contracts/pull/605/commits/167b8a9ddeff09555bb627b7a6419c1cffea36d5